generator/graphql-kotlin-federation/src/test/kotlin/com/expediagroup/graphql/generator/federation/execution/ServiceQueryResolverTest.kt

```diff
@@ -105,14 +105,14 @@ class ServiceQueryResolverTest {
             .coercing(CustomScalarCoercing()).build()
 
         private class CustomScalarCoercing : Coercing<CustomScalar, String> {
-            override fun parseValue(input: Any?): CustomScalar = CustomScalar(serialize(input))
+            override fun parseValue(input: Any): CustomScalar = CustomScalar(serialize(input))
 
-            override fun parseLiteral(input: Any?): CustomScalar {
+            override fun parseLiteral(input: Any): CustomScalar {
                 val customValue = (input as? StringValue)?.value ?: throw CoercingParseValueException("CustomScalar value is null")
```

Comment:
Is this exception still needed or should we change the message. Also this is a test so I suppose we could delete

Reply:
per Javadocs when unable to resolve a scalar then it should throw the `CoercingParseValueException`